src/main/java/de/butbetter/keksys/VATCalculator.java

```diff
@@ -0,0 +1,26 @@
+package de.butbetter.keksys;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+class VATCalculator implements Calculator {
+
+	private static final Logger log = LoggerFactory.getLogger(VATCalculator.class);
+
+	@Override
+	public Float calculateVAT(float price, float percent) {
+		float return_value = ((price / 100) * percent) + price;
+		log.info(price + " + " + percent + " = " + return_value);
+		return roundTo2DecimalPoints(return_value);
```

Comment:
Rounding to two decimals is fine for smaller values
But for really big values this is **fatal**:
Float Max = 3.4028235E38
But with this solution:
calculateVAT(Float.MaxValue,1) which should also be = 3.4028235E38, is just 2.1474836E7!

WE NEED TO FIND A DIFFERENT SOLUTION FOR THIS